PyInstaller/utils/win32/winmanifest.py

```diff
@@ -21,7 +21,7 @@
 # http://msdn.microsoft.com/en-us/library/dd408052%28VS.85%29.aspx
 #
 # Changelog:
-# 2009-12-17  fix: small glitch in toxml / toprettyxml methods (xml declaration wasn't replaced when a different encodig
+# 2009-12-17  fix: small glitch in toxml / toprettyxml methods (xml declaration wasn't replaced when a different encoding
 #                  than UTF-8 was used)
```

Comment:
You'll need to rewrap this to keep the linter happy since that one extra character put it above the 120 character line legth limit.
```suggestion
# 2009-12-17  fix: small glitch in toxml / toprettyxml methods (xml declaration wasn't replaced when a different
#                  encoding than UTF-8 was used)
```